metricbeat/module/docker/_meta/config.yml

```diff
@@ -1,13 +1,5 @@
 - module: docker
-  metricsets: ["container", "cpu", "diskio", "healthcheck", "info", "memory", "network"]
   hosts: ["unix:///var/run/docker.sock"]
```

Comment:
Out of the scope, but I think this should be the default.

Reply:
+1 on that.